docs/deployments/predictors.md

```diff
@@ -289,7 +311,7 @@ If your application requires additional dependencies, you can install additional
 ```python
 class ONNXPredictor:
     def __init__(self, onnx_client, config):
-        """Called once before the API becomes available. Performs setup such as downloading/initializing a vocabulary.
+        """(Required) Called once before the API becomes available. Performs setup such as downloading/initializing a vocabulary.
 
         Args:
             onnx_client: ONNX client which is used to make predictions. This should be saved for use in predict().
```

Comment:
We should add (required) labels to the init function args as well

Reply:
Makes sense!